docs/develop/auth.md

```diff
@@ -90,13 +91,21 @@ import (
 // AuthHandler can be named whatever you prefer (but must be exported).
 //encore:authhandler
 func AuthHandler(ctx context.Context, token string) (auth.UID, error) {
-    // if the token is invalid/expired you can return a custom error message
-    // using the code below.
-    return "", errs.B().Code(errs.Unauthenticated).Msg("auth token is invalid").Err()
+    return "", &errs.Error{
+        Code: errs.Unauthenticated,
+        Message: "invalid token",
+    }
 }
+
+
 ```
 
 
+<Callout type="important">
+Note that for security reasons you may not want to reveal too much information about why a request did not pass your auth checks. There are many subtle security considerations when dealing with authentication and we don't have time to go into all of them here. Whenever possible we recommend using a third-party auth provider; see [Using Firebase Authentication](../how-to/firebase-auth) for an example of how to do that.
+</Callout>
+
+
```

Comment:
Minor nitpick: can remove the extra newline here

Reply:
Okay